src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,30 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
+          <UserInfo user={user} />
+        </p>
```

Comment:
in some cases user can be null.
better to render this block conditionally
```suggestion
        {user && (
          <p>
            {' Posted by  '}
            <UserInfo user={user} />
          </p>
        )}
```